cmd/cli/kubectl-kyverno/commands/fix/doc.go

```diff
@@ -1,7 +1,6 @@
 package fix
 
-// TODO
-var websiteUrl = ``
+var websiteUrl = `https://kyverno.io/docs/kyverno-cli/#fix`
```

Comment:
Can you revert this change? As there is no section for the fix command in the docs.